doc/releases/changelog-0.34.0.md

```diff
@@ -49,62 +50,65 @@
 
-  There are two ways in which mid-circuit measurement statistics can be collected:
-
-  * By using arithmetic/binary operators. This can be through unary or binary operators as such:
-
-    ```python
-    import pennylane as qml
-
-    dev = qml.device("default.qubit")
-
-    @qml.qnode(dev)
-    def circuit(phi, theta):
-        qml.RX(phi, wires=0)
-        m0 = qml.measure(wires=0)
-        qml.RY(theta, wires=1)
-        m1 = qml.measure(wires=1)
-        return qml.expval(~m0 + m1)
-
-    print(circuit(1.23, 4.56))
-    ```
-    ```
-    1.2430187928114291
-    ```
-
-  * By using a list of mid-circuit measurement values:
-
-    ```python
-    import pennylane as qml
-
-    dev = qml.device("default.qubit")
-
-    @qml.qnode(dev)
-    def circuit(phi, theta):
-        qml.RX(phi, wires=0)
-        m0 = qml.measure(wires=0)
-        qml.RY(theta, wires=1)
-        m1 = qml.measure(wires=1)
-        return qml.sample([m0, m1])
-
-    print(circuit(1.23, 4.56, shots=5))
-    ```
-    ```
-    [[0 1]
-     [0 1]
-     [0 0]
-     [1 0]
-     [0 1]]
-    ```
-
-  This feature is supported on `default.qubit`, `default.qubit.legacy`, and `default.mixed`. To
-  learn more about which measurements and arithmetic operators are supported, refer to the
-  [Measurements](https://docs.pennylane.ai/en/stable/introduction/measurements.html) page and the
+  Mid-circuit measurement results can be composed using basic arithmetic operations and then
+  statistics can be calculated by putting the result within a PennyLane
+  [measurement](https://docs.pennylane.ai/en/stable/introduction/measurements.html) like
+  `qml.expval()`. For example:
+
+  ```python
+  import pennylane as qml
+
+  dev = qml.device("default.qubit")
+
+  @qml.qnode(dev)
+  def circuit(phi, theta):
+      qml.RX(phi, wires=0)
+      m0 = qml.measure(wires=0)
+      qml.RY(theta, wires=1)
+      m1 = qml.measure(wires=1)
+      return qml.expval(~m0 + m1)
+
+  print(circuit(1.23, 4.56))
+  ```
+  ```
+  1.2430187928114291
+  ```
+
+  Another option, for ease-of-use when using `qml.sample()`, `qml.probs()`, or `qml.counts()`, is to
+  provide a simple list of mid-circuit measurement results:
+
+  ```python
+  import pennylane as qml
+
+  dev = qml.device("default.qubit")
+
+  @qml.qnode(dev)
+  def circuit(phi, theta):
+      qml.RX(phi, wires=0)
+      m0 = qml.measure(wires=0)
+      qml.RY(theta, wires=1)
+      m1 = qml.measure(wires=1)
+      return qml.sample(op=[m0, m1])
+
+  print(circuit(1.23, 4.56, shots=5))
+  ```
+
+  ```
+  [[0 1]
+   [0 1]
+   [0 0]
+   [1 0]
+   [0 1]]
+  ```
+
+  Composite mid-circuit measurement statistics are supported on `default.qubit` and `default.mixed`.
+  To learn more about which measurements and arithmetic operators are supported, refer to the
+  [measurements page](https://docs.pennylane.ai/en/stable/introduction/measurements.html) and the
   documentation for [`qml.measure`](https://docs.pennylane.ai/en/stable/code/api/pennylane.measure.html).
```

Comment:
This link is not rendering correctly, as ReST does not supported nested formatting in links:

![image](https://github.com/PennyLaneAI/pennylane/assets/2959003/5542c600-e107-4f71-a070-16545176ae61)

Reply:
```suggestion
  [documentation for qml.measure](https://docs.pennylane.ai/en/stable/code/api/pennylane.measure.html).
```